style.css

```diff
@@ -3,3 +3,155 @@
   padding: 0;
   box-sizing: border-box;
 }
+
+:root {
+  --color-white: #f3f3f3;
+  --color-darkblue: #1b1b32;
+  --color-darkblue-alpha: rgba(27, 27, 50, 0.8);
+  --color-green: #37af65;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: var(--color-white);
+  margin: 0;
+}
+
+body::before {
+  content: '';
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: -1;
+  background: var(--color-darkblue);
+  background-image: url(https://c0.wallpaperflare.com/preview/811/935/645/adults-analysis-brainstorming-collaboration.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+h1 {
+  font-weight: 400;
+  line-height: 1.2;
+}
+
+p {
+  font-size: 1.125rem;
+}
+
+h1,
+p {
+  margin-top: 0;
+  margin-bottom: 0.5rem;
+}
+
+label {
+  display: flex;
+  align-items: center;
+  font-size: 1.125rem;
+  margin-bottom: 0.5rem;
+}
+
+.container {
+  width: 100%;
+  margin: 3.1rem auto 0 auto;
+}
+
+@media (min-width: 576px) {
+  .container {
+    max-width: 500px;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    max-width: 700px;
+  }
+}
+
```

Comment:
It is noticed that you use media queries for two different sizes, Kindly use one size that I would like to recommend 768px. 😄

Reply:
- [ ] Dear @MasumaJaffery, It's nice you have left your comment using GitHub inline comments feature. But please remember we are expected to always include an explanation, a "why", for each requested change. You have mentioned the use of `media queries for two different sizes` and suggested `768px` but never explain why this is better. We need to make sure the code owner understands why you are asking for changes. Please rephrase this comment in a way it explains why you are asking for the changes. 
> Feel free to edit this comment.

Reply:
I update the comment, hope this time it makes a clear concept.
Thank You!